injector_test.py

```diff
@@ -1700,6 +1700,67 @@ def function(a: int) -> 'InvalidForwardReference':
     assert get_bindings(function) == {'a': int}
 
 
+def test_gets_bindings_for_annotated_type_with_inject_decorator() -> None:
+    UserID = Annotated[int, 'user_id']
+
+    @inject
+    def function(a: UserID, b: str) -> None:
+        pass
+
+    assert get_bindings(function) == {'a': UserID, 'b': str}
+
+
+def test_gets_bindings_of_annotated_type_with_inject_annotation() -> None:
```

Comment:
All of these tests are probably welcomed (not familiar with the codebase, I have no idea), but it looks like codecov integration is currently broken. How about we set coveralls or codecov anew and start measuring the test coverage impact of code changes?